src/mixedmodel.jl

```diff
@@ -91,16 +91,16 @@ StatsAPI.nobs(m::MixedModel) = length(m.y)
 StatsAPI.predict(m::MixedModel) = fitted(m)
 
 function retbl(mat, trm)
-    return merge(
-        NamedTuple{(fname(trm),)}((trm.levels,)),
-        columntable(Tables.table(transpose(mat); header=Symbol.(trm.cnames))),
-    )
+    nms = (fname(trm), Symbol.(trm.cnames)...)
+    return Table(
+    [NamedTuple{nms}((l, view(mat, :, i)...),) for (i, l) in enumerate(trm.levels)]
+)
 end
 
 """
     raneftables(m::MixedModel; uscale = false)
 
-Return the conditional means of the random effects as a `NamedTuple` of columntables
+Return the conditional means of the random effects as a `NamedTuple` of `DictTable`s
```

Comment:
```suggestion
Return the conditional means of the random effects as a `NamedTuple` of Table.jl-compliant tables.

!!! note
    The API guarantee is only that there the NamedTuple contains Tables.jl tables and not on the particular concrete type of each table.
```

`DictTable` is technically a column table, so this is nonbreaking, but it's good to highlight the actual guarantee.

Reply:
Can you edit the suggestion to replace "Table.jl" with "Tables.jl" and, in the note, replace "only that there the" with "only that the"?

Reply:
@dmbates done